samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.Core.sln

```diff
@@ -0,0 +1,30 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26730.12
+MinimumVisualStudioVersion = 15.0.26730.12
```

Comment:
Run `projectStandards.linq` script

Reply:
Planning to run projectStandards on master once this is merged. There's lots of junk going on out there, far beyond this PR.